doc/releases/release-notes-2.5.rst

```diff
@@ -173,6 +173,10 @@ Drivers and Sensors
 
 * USB
 
+  * Added support for operate only on the primary image by the DFU class which,
```

Comment:
```suggestion
  * Support for USB DFU with single slot mode.
  * Support to use USB DFU within MCUBoot with single application slot mode.
```

Reply:
single slot mode is notion which belongs to MCUboot. Zephyr can provide single image partition.

Reply:
@emob-nordic got better phrase from tech writer